tests/e2e/secretsencryption/Vagrantfile

```diff
@@ -44,6 +44,7 @@ def provision(vm, role, role_num, node_num)
   if vm.box.to_s.include?("microos")
     vm.provision 'k3s-reload', type: 'reload', run: 'once'
   end
+  vm.provision 'k3s-autocomplete', type: 'shell', inline: "k3s completion -i bash"
```

Comment:
TODO: integrate this directly into the K3s Vagrant plugin, so all our E2E tests just have autocomplete on the nodes, makes debugging tests faster.